src/inspector_agent.cc

```diff
@@ -567,12 +578,30 @@ void AgentImpl::WaitForDisconnect() {
                            v8::ReadOnly).FromJust();                          \
   } while (0)
 
+void AddCommandLineAPIMethod(const v8::FunctionCallbackInfo<v8::Value>& info) {
```

Comment:
My C++ is a bit rusty, so the following might be just me missing one of the "things `static` can mean": It's no longer a class member but a top-level function as of the latest commit.